presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -147,6 +147,29 @@ std::shared_ptr<connector::hive::LocationHandle> toLocationHandle(
       toTableType(locationHandle.tableType));
 }
 
+dwio::common::FileFormat toStorageFormat(
```

Comment:
It looks a bit strange for a method called toStorageFormat to return a FileFormat. Can this be called toFileFormat instead ?

Reply:
Good point. Fixed!